### DISCUSSION TITLE
Cannot reach Turkish lookup table

### DISCUSSION THREAD

**Discussion by canberkarc** [explosion/spaCy#11142] [Help: Coding & Implementations] [answered]
I have installed spacy-lookups-data 1.0.3 and it contains Turkish lookup table,too but I cannot see any table when I check tables.

``
import spacy
from spacy.language import Language
from spacy.lookups import Table
from negation_turkish import Negex
from spacy.lang.tr import Turkish

tr = spacy.blank("tr")

lemmatizer = tr.add_pipe("lemmatizer")
lemmatizer.initialize()
tr.add_pipe("sentencizer")
tr.add_pipe("negex", last = True)
all_stopwords = tr.Defaults.stop_words

tr.vocab.lookups.tables
``

What can be the problem? Thanks in advance!

**Comment by adrianeboyd** [comment, marked as answer]
In spacy v3 the lemmatizer lookup tables have moved from `nlp.vocab.lookups` to the lemmatizer component: `nlp.get_pipe("lemmatizer").lookups`

**Reply by canberkarc** [reply]
Thank you for your answer!
